packages/lb-components/src/components/pointCloudView/index.tsx

```diff
@@ -49,10 +49,21 @@ const PointCloudView: React.FC<IProps> = ({
   checkMode,
   intelligentFit,
 }) => {
-  const { globalPattern } = useContext(PointCloudContext);
+  const { globalPattern, setGlobalPattern } = useContext(PointCloudContext);
 
   const basicInfo = jsonParser(currentData.result);
-  const { toolInstanceRef } = useCustomToolInstance({ basicInfo });
+  const { toolInstanceRef, clearToolInstance } = useCustomToolInstance({ basicInfo });
+
+  useEffect(() => {
+    clearToolInstance()
```

Comment:
It's not a good place to clear data which depend on useEffect. You can clear data after patternChange

Reply:
done